src/api/auth.js

```diff
@@ -0,0 +1,50 @@
+import jwt from 'jsonwebtoken';
+import { logger } from '../logger.js';
+
+import { configureServices } from '../config.js';
+
+const { usersService } = configureServices();
+
+export const checkToken = (req, res, next) => {
+    const token = req.headers.authorization?.split(' ')?.[1];
+
+    if (!token) {
+        return res.status(401).send({ message: 'No token provided' });
+    }
+
+    const secret = process.env.SECRET;
+
+    try {
+        jwt.verify(token, secret);
+    } catch (error) {
+        logger.warn({
+            message: error.message
+        });
+        res.status(403).send({ message: error.message });
+    }
+
+    return next();
+};
+
+export const handleLogin = async (req, res) => {
+    const { login, password } = req.body;
+
+    if (!login || !password) {
+        return res.status(401).send({
+            message: 'Bad username/password combination'
+        });
+    }
+
+    const user = await usersService.getByLogin(login);
+    if (user.password !== password) {
```

Comment:
do you store password in raw format? That's not correct, you should store only password hash and verify inc password against that hash, Check https://www.npmjs.com/package/bcrypt

Reply:
Added password hashing